src/main/java/Cy.java

```diff
@@ -0,0 +1,352 @@
+import commands.Deadline;
+import commands.Event;
+import commands.Task;
+import commands.Todo;
+import exceptions.IllegalCommandException;
+import exceptions.IllegalEmptyException;
+import exceptions.IllegalKeywordException;
+import exceptions.IllegalTaskException;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Cy {
```

Comment:
your main class is doing too many things. please extract related methods to additional classes. see A-MoreOOP for guidelines.